### PR TITLE
Add network policy

### PR DESCRIPTION
**Pull request by glitchcrab**
Towards giantswarm/giantswarm/issues/4383

### CODE REVIEW

File: helm/credentiald-chart/values.yaml [hunk 1]
```diff
@@ -1,8 +1,30 @@
...
+image:
+  registry: "quay.io"
+  repository: "giantswarm/credentiald"
+  tag: "[[ .SHA ]]"
 pod:
   user:
     id: 1000
   group:
     id: 1000
+resource:
```
Comment (corest) [inline helm/credentiald-chart/values.yaml]: How is this related to network policies?

Comment (glitchcrab) [reply helm/credentiald-chart/values.yaml]: it's not exactly, but I was trying to standardise values.yml at the same time. I can do it in a separate PR if necessary. Which do you prefer?
